samples/server/petstore/python-flask-python2/openapi_server/openapi/openapi.yaml

```diff
@@ -602,7 +602,7 @@ components:
         petId: 6
         quantity: 1
         id: 0
-        shipDate: 2000-01-23T04:56:07.000+00:00
```

Comment:
Should this be loaded as a string -> datetime whether or not the yaml has quotes?
Can you add a test showing how this behavior has changed with your update in python code?

Reply:
I might. I just don't think this test will have much value - it would simply fail with an exception at setup.

Since the way the API definitions are loaded is described in _flask_, this would be essentially testing somewhere in between flask's test utils and python's `copy.deepcopy` (which doesn't handle objects well). So in my opinion, this has little to do with the **openapi-generator**.

Reply:
Most importantly, currently `master` is failing because of this ;)

Reply:
It turned to be a good idea, after all - python2 was affected, whilst python3 was not. Added tests to highlight that.